### PR TITLE
Aet 162 add kong component

### PR DESCRIPTION
**Pull request by lordmallam**
#### What does this PR do?
Adds Kong as a component to the docker compose structure reachable on kong admin: `http://kong:8001` kong apis: `http://kong:8000`

#### How to test
Run `docker-compose up --build kernel`
Kong container is started with an ephemeral `kong-migration` container that handles migrations to the db
Kong admin should be reachable via `http://kong:8001` from within the `kernel` container or `kong.aether.local:8001` remotely (this requires that `kong.aether.local` be added to the /etc/host file)

#### Related JIRA ticket
[AET-162](https://jira.ehealthafrica.org/browse/AET-162)

### CODE REVIEW

**Review by shawnsarwar — APPROVED** [review]
Looks good. I can hit it on localhost:8001

File: docker-compose-base.yml [hunk 1]
```diff
@@ -11,16 +11,38 @@
 # See more in: https://docs.docker.com/compose/extends/
 # ------------------------------------------------------------------------------
 
-version: "2"
+version: "2.1"
```
Comment (admbtlr) [inline docker-compose-base.yml:14]: This is probably a breaking change, isn't it? If so, this should be `3.0`

**Review by admbtlr — CHANGES_REQUESTED** [review]

File: docker-compose-base.yml (hunk 1 above)
Comment (MrPink) [reply docker-compose-base.yml:14]: This is the docker-compose version, what does the minor release fix @lordmallam

Comment (MrPink) [reply docker-compose-base.yml:14]: This is the docker-compose version, what does the minor release fix @lordmallam

Comment (admbtlr) [reply docker-compose-base.yml:14]: D'oh! Ignore me...

Comment (admbtlr) [reply docker-compose-base.yml:14]: @MrPink I'm going to hit approve, can you please hit "Request changes" until @lordmallam answers your question?

**Review by admbtlr — APPROVED** [review]

File: docker-compose-base.yml [hunk 2]
```diff
@@ -11,16 +11,38 @@
...
   # ---------------------------------
-  # Database + NGINX containers
+  # Database + Kong + NGINX containers
   # ---------------------------------
 
   postgres-base:
     image: postgres:9.6
+    environment:
+      - POSTGRES_USER=kong
+      - PSOTGRES_DB=kong
```
Comment (quesebifurcan) [inline docker-compose-base.yml]: spelling -- `PSOTGRES_DB` -> `POSTGRES_DB`?

File: docker-compose-base.yml (hunk 1 above)
Comment (lordmallam) [reply docker-compose-base.yml:14]: @MrPink `2.1` allows the use of `depends_on > condition`  for health checks

**Review by MrPink — APPROVED** [review]

File: docker-compose-base.yml (hunk 1 above)
Comment (MrPink) [reply docker-compose-base.yml:14]: @lordmallam that's awesome :-) finally!

**Pull request merged**